LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 ngi-nix
```

Comment:
```suggestion
Copyright (c) 2023 the contributors to NGIpkgs
```

I'd prefer something that's a little more specific than just 'ngi-nix'. Even though I understand we have to be sufficiently vague.

Reply:
I just dropped the GitHub template, thanks for the fixup!

```suggestion
Copyright (c) 2023 NGIpkgs contributors
```